features/admin.userstores.v1/pages/user-stores.tsx

```diff
@@ -222,6 +228,50 @@ const UserStores: FunctionComponent<UserStoresPageInterface> = (
         setFilteredUserStores(userStores);
     };
 
+    const addUserDropdownTrigger: ReactElement = (
+        <PrimaryButton
+            data-componentid={ `${ testId }-add-user-button` }
+            data-testid={ `${ testId }-add-user-button` }
+        >
+            <Icon name="add"/>
+            { t("userstores:pageLayout.list.primaryAction") }
+            <Icon name="dropdown" className="ml-3 mr-0"/>
+        </PrimaryButton>
+    );
+
+    const getAddUserOptions = (): DropdownItemProps[] => {
+        const dropDownOptions: DropdownItemProps[] = [];
+
+        dropDownOptions.push({
+            "data-componentid": `${testId}-add-user-dropdown-item`,
+            key: 1,
+            text: t("userstores:pageLayout.list.newUserStoreDropdown.connectDirectly"),
+            value: UserStoreTypes.DIRECT
+        });
+        dropDownOptions.push({
+            "data-componentid": `${testId}-bulk-import-users-dropdown-item`,
+            "data-testid": `${testId}-bulk-import-users-dropdown-item`,
```

Comment:
Check whether we need both `componentid` and `testid`.

Reply:
This is added for backward compatibility.